src/tasks/containers/TasksPage.tsx

```diff
@@ -67,10 +67,6 @@ class TasksPage extends PureComponent<Props, State> {
   constructor(props: Props) {
     super(props)
 
-    if (!props.showInactive) {
-      props.setShowInactive()
-    }
-
```

Comment:
this code runs every time TasksPage mounts. Which is exactly what's re-setting inactive tasks to show after user has explicitly turned off the `show inactive` toggle. 
I think the purpose here was to show inactive tasks on initial load. But since we have a default state (set to true) saved in redux, there is no need to set it to true again here.